gateway/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM node:18-slim AS deps
 # Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
-RUN apk add --no-cache libc6-compat
+RUN apt install libc6
```

Comment:
Thanks for the PR :+1: 
Still given the comment above, still not very confident it won't break anything...
Do you know the difference between `libc6` and `libc6-compat`?

Reply:
I don't think, it will break something because libc6-compat is only set of lib compatible with glib since alpine use musl no?